fixtures/anchorEpisodeAggregatedPerformance.json

```diff
@@ -0,0 +1,52 @@
+{
+  "provider": "anchor",
+  "version": 1,
+  "retrieved": "2023-05-05T16:20:59.449713",
+  "meta": {
+    "show": "123abcde",
+    "endpoint": "aggregatedPerformance",
+    "web_episode_id": "foo1234"
```

Comment:
this should be `episodeId` as in the other endpoints?

Reply:
you're right. fixed.